sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala

```diff
@@ -54,11 +56,32 @@ case class Statistics(
 
   /** Readable string representation for the Statistics. */
   def simpleString: String = {
-    Seq(s"sizeInBytes=$sizeInBytes",
-      if (rowCount.isDefined) s"rowCount=${rowCount.get}" else "",
+    Seq(s"sizeInBytes=${format(sizeInBytes, isSize = true)}",
+      if (rowCount.isDefined) s"rowCount=${format(rowCount.get, isSize = false)}" else "",
       s"isBroadcastable=$isBroadcastable"
     ).filter(_.nonEmpty).mkString(", ")
   }
+
+  /** Print the given number in a readable format. */
+  def format(number: BigInt, isSize: Boolean): String = {
```

Comment:
I'll try to use that method in combination with current logic, thanks for reminding